### ISSUE TITLE
training time and how to enhance the network? 

### ISSUE THREAD

**Issue by mhfatemi** [Lornatang/Real_ESRGAN-PyTorch#4]
hello and thank you for your great work
how long did take training of your network?
is it possible to reduce the number of RRDB blocks??
is it possible to change loss functions??

**Comment by Lornatang** [comment]
1. 14 days RRDB + 8 days ESRGAN ~= 21 days.
2. Can reduce RRDB, e. 8, 16;
3. I am also trying to modify Loss recently, can you leave a contact information to discuss together~

**Comment by mhfatemi** [comment]
_> 1. 14 days RRDB + 8 days ESRGAN ~= 21 days.
> 2. Can reduce RRDB, e. 8, 16;
> 3. I am also trying to modify Loss recently, can you leave a contact information to discuss together~_

can't use of pretrained ".path" files if we use 8 or 16 RRDB. I'm correct??
my mail address is "fatemi.m@ec.iut.ac.ir" 
I'm so glad if you share me your progress in super resolution

**Comment by Lornatang** [comment]
It is also possible, in this project some weights will be loaded automatically

**Comment by Lornatang** [comment]
finetune it ~
